docs/running-queues.md

```diff
@@ -65,11 +65,12 @@ This way, worker will consume jobs with the `low` priority and then with `high`.
 
 ### Running many instances of the same queue
 
-As mentioned above, sometimes we may want to have multiple instances of the same command running at the same time. The queue is safe to use in that scenario with all databases if you keep the `skipLocked` to `true` in the config file. Only for SQLite3 driver this setting is not relevant.
+As mentioned above, sometimes we may want to have multiple instances of the same command running at the same time. The queue is safe to use in that scenario with all databases if you keep the `skipLocked` to `true` in the config file.
+Only for SQLite3, PHPRedis and Predis driver, this setting is not relevant.
```

Comment:
This is not clear. Do you mean SQLite3, phpredis and redis drivers support this setting? Or vice versa?

Reply:
Good point. Updated - hopefully not it's more clear.